docs/layouts/shortcodes/generated/core_configuration.html

```diff
@@ -182,6 +182,12 @@
             <td>Double</td>
             <td>Define the default false positive probability for bloom filters.</td>
         </tr>
+        <tr>
+            <td><h5>orc.compression.per.level</h5></td>
```

Comment:
`file.compression.per.level`? This also work for parquet too.

Reply:
yes,I should add conf for parquet

Reply:
we can not define the key to `file.compression.per.level` ,  because we need to get the orc write properties by the `orc` prefix for `OrcWriterFactory#writerProperties`,  so I add an parquet key `parquet.compression.per.level`